src/python_testing/TC_TSTAT_4_2.py

```diff
@@ -269,7 +269,9 @@ async def test_TC_TSTAT_4_2(self):
             await self.send_atomic_request_begin_command()
 
             # Write to the presets attribute after calling AtomicRequest command
-            await self.write_presets(endpoint=endpoint, presets=new_presets)
+            test_presets = copy.deepcopy(new_presets)
```

Comment:
We should test both cases: null on a preset matching an existing handle, and null on a preset with a null handle.

Reply:
OK, added to the above test for the other case